.travis.yml

```diff
@@ -1,11 +1,8 @@
-sudo: required
-dist: trusty
-
 language: go
+os: linux
```

Comment:
I think you can specify multiple OSes here, should we run these tests on all platforms?

```
os:
  - linux
  - osx
  - windows
```

Reply:
I tried this, but travis always has a backlog of OSX jobs (its at ~800 now).  I think its probably best to just test on one platform for now.